enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/util/ToolbarButton/ToolbarButton.tsx

```diff
@@ -12,16 +12,16 @@ function _ToolbarButton(
     ...buttonProps
   }: {
     label: ReactNode;
-    icon: IconName;
-    isHighlighted: boolean;
+    icon?: IconName;
+    isHighlighted?: boolean;
```

Comment:
I've checked that we always pass these 2 props, also with reverting to make them required still pass `yarn type-check` so I suggest reverting it.

```suggestion
    icon: IconName;
    isHighlighted: boolean;
```

This will allows us to revert the change below as well.

Reply:
I'm actually going to use this component in the next two PRs without `icon` since I want the `Save` button to use the same styles. I _could_ make a new component but I figured it would be easier to reuse this in the future. 

Would you prefer that I change it here, and then make the change in the `Save` button PR?

Reply:
That will make the change easier to understand for each PR, so I prefer that yeah. It's hard to understand a PR with outside context.